From: Martin Klapetek <mklapetek@kde.org>
Date: Thu, 17 Sep 2015 16:11:30 +0000
Subject: [kaccounts] Emit configUiReady() when config UI is ready
X-Git-Url: http://quickgit.kde.org/?p=ktp-accounts-kcm.git&a=commitdiff&h=07c7cc5309e11171b2e552678d58257ccfc44e42
---
[kaccounts] Emit configUiReady() when config UI is ready

Should prevent the crash when clicking the Configure button

(requires kaccounts-integration 15.08.2)

BUG: 351785
FIXED-IN: 15.08.2
---


--- a/plugins/kaccounts/kaccounts-ui-provider.cpp
+++ b/plugins/kaccounts/kaccounts-ui-provider.cpp
@@ -106,10 +106,10 @@
 
     if (d->type == KAccountsUiPlugin::ConfigureAccountDialog) {
         if (d->accountManager->isReady()) {
-            Q_EMIT uiReady();
+            Q_EMIT configUiReady();
         } else {
             // let's wait for AM to become ready first
-            connect(d->accountManager->becomeReady(), &Tp::PendingOperation::finished, this, &KAccountsUiProvider::uiReady);
+            connect(d->accountManager->becomeReady(), &Tp::PendingOperation::finished, this, &KAccountsUiProvider::configUiReady);
         }
     }
 }

